src/mlpack/tests/gan_test.cpp

```diff
@@ -215,7 +215,8 @@ BOOST_AUTO_TEST_CASE(GANMNISTTest)
       discriminatorPreTrain, multiplier);
 
   Log::Info << "Training..." << std::endl;
-  double objVal = gan.Train(optimizer);
+  std::stringstream stream;
+  double objVal = gan.Train(optimizer, ens::PrintLoss(stream));
```

Comment:
We should check that the callback works, e.g. by checking if the length of the `stream` parameter isn't empty.

Reply:
done.